### BIG differences. This is diff with two wrong files, CLUS "Function CONS"
### and CLHS topic selection http://www.lispworks.com/documentation/lw50/CLHS/Body/a_cons.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:cons.txt	2017-04-27 22:02:42.000000000 +0100
+++ saved-pages-clus/cl:functions:cons.txt	2017-04-27 22:02:42.000000000 +0100
@@ -1,4 +1,47 @@
-CONS
-Please select which reference to CONS you intended
+Function CONS

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Syntax
+ cons object-1 object-2 → cons
+Arguments and values
+ object-1 - an object.
+ object-2 - an object.
+ cons - a cons.
+Description
+Creates a fresh cons, the car of which is object-1 and the cdr of which is object-2.
+Examples
+(cons 1 2)
+→
+(1 . 2)
+(cons 1 nil)
+→
+(1)
+(cons nil 2)
+→
+(NIL . 2)
+(cons nil nil)
+→
+(NIL)
+(cons 1 (cons 2 (cons 3 (cons 4 nil))))
+→
+(1 2 3 4)
+(cons 'a 'b)
+→
+(A . B)
+(cons 'a (cons 'b (cons 'c '())))
+→
+(A B C)
+(cons 'a '(b c d))
+→
+(A B C D)
+Side Effects
+None.
+Affected By
+None.
+Exceptional Situations
+None.
+See also
+Function LIST
+Example Implementation
+To be done.
+Notes
+If object-2 is a list, cons can be thought of as producing a new list which is like it but has object-1 prepended.
\ No newline at end of file
